action.yml

```diff
@@ -70,6 +70,9 @@ inputs:
   dockerfile_path:
     description: 'Path to the Dockerfile of the scanned docker image'
     required: false
+  var_file:
+    description: 'Use custom values file when scanning a Helm Chart'
```

Comment:
From checkov --help
```
Variable files to load in addition to the default
                        files (see https://www.terraform.io/docs/language/valu
                        es/variables.html#variable-definitions-tfvars-
                        files).Currently only supported for source Terraform
                        (.tf file), and Helm chart scans.Requires using
                        --directory, not --file.

```

Reply:
@kartikp10, I updated the description based on your comment.